### PR TITLE
add minimal test whether pty logging works

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@mailbox.org>

### CODE REVIEW

**Comment by brauner** [comment]
jenkins: test this please

**Comment by brauner** [comment]
download failure
jenkins: test this please

File: src/tests/lxc-test-lxc-attach [hunk 1]
```diff
@@ -175,6 +175,14 @@ if [ "$outcontent" != "busy" ] || [ -z "$errcontent" ]; then
         FAIL " echo 'hostname; rm' | lxc-attach -n busy > $out 2> $err"
 fi
 
+# Test whether logging pty output to a file works.
+lxc-attach -n busy -L /tmp/ptylog -- hostname
+trap "rm -f /tmp/ptylog" EXIT INT QUIT PIPE
+logcontent=$(cat /tmp/ptylog)
+if [ -e /tmp/ptylog ] && [ -z "$logcontent" ]; then
```
Comment (hallyn) [inline src/tests/lxc-test-lxc-attach]: Why do you not FAIL is ! -e /tmp/ptylog?

File: src/tests/lxc-test-lxc-attach [hunk 2]
```diff
@@ -175,6 +175,14 @@ if [ "$outcontent" != "busy" ] || [ -z "$errcontent" ]; then
         FAIL " echo 'hostname; rm' | lxc-attach -n busy > $out 2> $err"
 fi
 
+# Test whether logging pty output to a file works.
+lxc-attach -n busy -L /tmp/ptylog -- hostname
+trap "rm -f /tmp/ptylog" EXIT INT QUIT PIPE
```
Comment (hallyn) [inline src/tests/lxc-test-lxc-attach]: What about removing /tmp/ptylog now?

**Comment by brauner** [comment]
I'm not happy with this.

**Pull request closed without merging**